AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -148,55 +168,55 @@ - (BOOL)canPerformAction:(SEL)action withSender:(id)sender
 
 - (CGFloat)alpha
 {
-  _bridge_prologue;
+  _bridge_prologue_read;
   return _getFromViewOrLayer(opacity, alpha);
 }
 
 - (void)setAlpha:(CGFloat)newAlpha
 {
-  _bridge_prologue;
+  _bridge_prologue_write;
   _setToViewOrLayer(opacity, newAlpha, alpha, newAlpha);
 }
 
 - (CGFloat)cornerRadius
 {
-  _bridge_prologue;
+  _bridge_prologue_read;
   return _getFromLayer(cornerRadius);
 }
 
 - (void)setCornerRadius:(CGFloat)newCornerRadius
 {
-  _bridge_prologue;
+  _bridge_prologue_write;
   _setToLayer(cornerRadius, newCornerRadius);
```

Comment:
My first thought upon seeing the read and write terminology was - what about get and set?  I think read and write is probably better, as they are more prominent and visibly different as they probably should be.